mergometer.gemspec

```diff
@@ -34,6 +34,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "hirb"
   spec.add_dependency "octokit"
   spec.add_dependency "progress_bar"
+  spec.add_dependency "gruff"
```

Comment:
I would really question the addition of such a gem to `mergometer`.

Is it this gem's role to output images and PDFs, or the job of an application (or other gem) that uses `mergometer`?
Maybe we want a separate `mergometer-graphs` gem?

Reply:
Depends on what we want this gem to be able to do, I guess. Creating subgems sounds nice though!